apps/src/code-studio/pd/workshop_dashboard/components/workshop_management.jsx

```diff
@@ -25,9 +26,22 @@ const WorkshopManagement = React.createClass({
     };
   },
 
+  componentWillMount() {
+    if (this.props.showSurveyUrl) {
+      this.permission = new Permission();
+
+      const surveyBaseUrl = this.permission.isOrganizer ? "/organizer_survey_results" : "/survey_results";
+      const surveyUrl = this.props.showSurveyUrl && `${surveyBaseUrl}/${this.props.workshopId}`;
+
+      this.setState({
+        surveyUrl: surveyUrl
+      });
```

Comment:
I thought we earlier talked about how we wanted WorkshopManagement to make the determination here, instead of go off something provided